0003-use-of-single-signon.md

```diff
@@ -0,0 +1,78 @@
+# Provisions for the use of single sign on when accessing treetracker web applications
```

Comment:
We need to be careful with the term _Single Sign-On_ when talking about Keycloak, which uses SSO to mean signing into Keycloak allows access to multiple applications.
For what we're discussing here, Keycloak uses the terms _Identity Brokering and Social Login_.

Reply:
Updated ADR to use terminology consistent with keycloak docs